Source/FluentSpreadsheets.ClosedXML/Rendering/ClosedXmlComponentRenderer.cs

```diff
@@ -0,0 +1,17 @@
+using FluentSpreadsheets.ClosedXML.Visitors;
+using FluentSpreadsheets.Rendering;
+using FluentSpreadsheets.Visitors;
+
+namespace FluentSpreadsheets.ClosedXML.Rendering;
+
+public class ClosedXmlComponentRenderer : IComponentRenderer<ClosedXmlRenderCommand>
+{
+    public Task RenderAsync(ClosedXmlRenderCommand command, CancellationToken cancellationToken = default)
+    {
+        var handler = new ClosedXmlHandler(command.Worksheet);
+        var visitor = new ComponentVisitor<ClosedXmlHandler>(new Index(1, 1), handler);
```

Comment:
Why exactly are we passing new Index every time?

Reply:
it is parametrisation. component visitor lets you customise a starting point of rendered component. as index is a struct, there is no difference between storing it in a field and pass to visitor's constructor, define a constructor with default index, or the way it is now.

Reply:
K, I just think it'll be cleaner with one more constructor